docs/source/tutorials/parsl-rp/parsl-rp.ipynb

```diff
@@ -0,0 +1,276 @@
+    "The following example application shows the execution of MP2 geometry optimization followed by a CCSD(T) energy evaluation at the converged geometry. A Dunning correlation-consistent triple-zeta basis is used. The default of Cartesian basis functions must be overridden using the keyword spherical on the BASIS directive. The 1s core orbitals are frozen in both the MP2 and coupled-cluster calculations (note that these must separately specified)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52f78a41",
+   "metadata": {},
+   "source": [
+    "First, we need to write the `nwchem` example to a file so we can use it provide it as an input to the `nwchem` executable later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6980a500",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input = \"\"\"\n",
+    "start n2\n",
+    "\n",
+    "geometry\n",
+    "  symmetry d2h\n",
+    "  n 0 0 0.542\n",
+    "end\n",
+    "\n",
+    "basis spherical\n",
+    "  n library cc-pvtz\n",
+    "end\n",
+    "\n",
+    "mp2\n",
+    "  freeze core\n",
+    "end\n",
+    "\n",
+    "task mp2 optimize\n",
+    "\n",
+    "ccsd\n",
+    "  freeze core\n",
+    "end\n",
+    "\n",
+    "task ccsd(t)\n",
+    "\"\"\"\n",
+    "nwchem_input = '{0}/{1}'.format(os.getcwd(), 'mp2_optimization.nw')\n",
+    "with open(nwchem_input,\"w+\") as f:\n",
+    "    f.writelines(input)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39d40bda",
+   "metadata": {},
+   "source": [
+    "Now, we import parsl Python module in our application and the RadicalPilotExecutor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "77692350",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-11-22T12:17:57.635805Z",
+     "start_time": "2022-11-22T12:17:57.340827Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import parsl\n",
+    "import radical.pilot as rp\n",
+    "\n",
+    "from parsl.config import Config\n",
+    "from parsl.app.app import python_app, bash_app\n",
+    "from parsl.executors import RadicalPilotExecutor"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1290100d",
+   "metadata": {},
+   "source": [
+    "Set the path of RadicalPilotExecutor configuration file. The rpex.cfg is a configuraion file that describes the resource\n",
+    "distribuation between functions and executables for RadicalPilotExecutor and pass it to Parsl dataflow configuration with RadicalPilotExecutor to load it. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fde1d4f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rpex_cfg = 'rpex.cfg'\n",
+    "config = Config(\n",
+    "    executors=[RadicalPilotExecutor(\n",
+    "        rpex_cfg=rpex_cfg, bulk_mode=True,\n",
+    "        resource='local.localhost', login_method = 'local',\n",
+    "        walltime=30, managed= True, cores= 4\n",
+    "    )])\n",
```

Comment:
I would propose to show config within notebook similar to `nwchem` input
```python
config = """
{
    "cpn"            :    1,
    "gpn"            :    0,
    "n_masters"      :    1,
    "n_workers"      :    1,
    "masters_pn"     :    1,
    "nodes_pw"       :    1,
    "nodes_rp"       :    0,
    "tasks_rp"       :    1,
    "nodes_agent"    :    1,

    # use or create an existing env
    "pilot_env"      : {},

    "master_descr": {
        "named_env"   : "ve_rpex",
        "executable"  : "python3 rpex_master.py",
        "ranks"       : 1
    },

    "worker_descr": {
        "named_env"   : "ve_rpex",
        "worker_class": "MyWorker",
        "worker_file" : "./rpex_worker.py"
    }
}
"""
with open('config_rpex.json', 'w') as fd:
    fd.write(config)
...
config = Config(
    executors=[RadicalPilotExecutor(
        rpex_cfg='config_rpex.json', bulk_mode=True,
        resource='local.localhost', login_method = 'local',
        walltime=30, managed= True, cores= 4
    )])
```

Reply:
fixed